docs/build/avalanchego-apis/info.md

```diff
@@ -232,6 +232,59 @@ curl -X POST --data '{
 }
 ```
 
+
+### info.getVMs
+
+Get the virtual machines installed on this node.
+
+```text
```

Comment:
I don't know the exact reason but we use `cpp` syntax for these signatures. We should definitely take another pass on these in-line code types...

Reply:
Updated for consistency, but we should be using text imo. We can fix it in a separate pr.

Reply:
yeah and I'm afraid it might be a HUUGE PR.